types/pdfmake/index.d.ts

```diff
@@ -165,8 +165,13 @@ declare module 'pdfmake/build/pdfmake' {
         pageBreak?: 'before' | 'after';
         alignment?: Alignment;
         table?: Table;
-        ul?: Content[];
-        ol?: Content[];
+        ul?: Array<string | Content>;
+        ol?: Array<string | Content>;
+        [additionalProperty: string]: any;
+    }
+
+    interface ContentStack {
```

Comment:
I don't understand why we need the new `ContactStack`. The data model is very flexible and it should be sufficient to have the `stack` property on `Content`.

Can you explain what you try to accomplish with this new interface? Was this a result of your debugging?
Thank you for the effort!

Reply:
The type checking inside the test file was complaining about the content of the `stack` key, so I did that in order to make use of the existing `Content` structure inside the `stack` key name without affecting anything else.

If you check out the branch from my changes and remove that `interface`, you should see a type mismatch error show up.

Unless I'm mistaken and there's actually a better way to handle that?